src/components/DataRecordingGridItem.tsx

```diff
@@ -94,13 +98,29 @@ const DataRecordingGridItem = ({
                     handleDeleteRecording(idx);
                   }}
                 />
-                <RecordingGraph
-                  data={recording.data}
-                  role="image"
-                  aria-label={intl.formatMessage({
-                    id: "recording-graph-label",
-                  })}
-                />
+                {(!flags.fingerprints ||
+                  view === DataSamplesView.Graph ||
+                  view === DataSamplesView.GraphAndDataFeatures) && (
+                  <RecordingGraph
+                    data={recording.data}
+                    role="image"
+                    aria-label={intl.formatMessage({
+                      id: "recording-graph-label",
+                    })}
+                  />
+                )}
+                {flags.fingerprints &&
+                  (view === DataSamplesView.DataFeatures ||
+                    view === DataSamplesView.GraphAndDataFeatures) && (
+                    <RecordingFingerprint
+                      data={recording.data}
+                      role="image"
+                      gestureName={data.name}
+                      aria-label={intl.formatMessage({
+                        id: "recording-fingerprint-label",
```

Comment:
Understandable as there isn't a clear way to represent this data non-visually, at least for the purposes of easy comparison as this feature intends.

Please can you add this to your screenreader notes so we can pick up and consider an alternative?

Reply:
Makes sense, added it